tests/test_theory_parent.py

```diff
@@ -129,18 +125,12 @@ def test_instantiate(modeling_data):
     
     beta_s_square_test = compute_beta_s_square_mean(z_cl, 1000., mod.cosmo)
     beta_s_test = compute_beta_s_mean(z_cl, 1000., mod.cosmo)   
-    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, z_src, 'applegate14', beta_s_mean, beta_s_square_mean)
+    reduced_shear = mod.eval_reduced_tangential_shear(r_proj, z_cl, None, 'distribution', 'applegate14')
```

Comment:
first do `from clmm.z_distributions import chang2013, desc_srd` in the beginning of the file. Then changing `None` to `chang13` will fix the test.